.opspec/release/to-ghcr/op.yml

```diff
@@ -0,0 +1,95 @@
+name: to-docker
+description: releases opctl to [docker hub](https://hub.docker.com/r/opctl/opctl/)
+inputs:
+  github:
+    object:
+      constraints:
+        properties:
+          accessToken:
+            minLength: 1
+            type: string
+            writeOnly: true
+          username:
+            minLength: 1
+            type: string
+        required: [accessToken, username]
+      description: configuration required to interact w/ github
+  version:
+    string:
+      constraints: { format: semver }
+      description: version being published
+  dockerSocket:
+    socket:
+      description: docker socket
+    default: /var/run/docker.sock
+outputs:
+  alreadyPublished:
+    number:
```

Comment:
nit) opinionated but boolean would be more idiomatic IMHO (& slightly simpler e.g. no need to specify constraints)